src/views/Home/components/CakeDataRow.tsx

```diff
@@ -75,9 +73,9 @@ const CakeDataRow = () => {
       const tokenDataResultRaw = await multicallv2(cakeAbi, [totalSupplyCall, burnedTokenCall], {
         requireSuccess: false,
       })
-      const tokenDataResult = tokenDataResultRaw.flat()
-      setTotalSupply(tokenDataResult[0] ? getBalanceNumber(new BigNumber(tokenDataResult[0].toString())) : 0)
-      setBurnedBalance(tokenDataResult[1] ? getBalanceNumber(new BigNumber(tokenDataResult[1].toString())) : 0)
+      const [totalSupply, burned] = tokenDataResultRaw.flat()
+      setCakeSupply(totalSupply ? +formatBigNumber(totalSupply.sub(burned)) : 0)
```

Comment:
If burned comes null (which sometimes happens I think), what would be the output?